README.md

```diff
@@ -161,7 +159,6 @@ For example, if you want to use an environment variable to set the `name` parame
 |`KAFKA_ADMIN-CLIENT-TIMEOUT` | Kafka API timeout in ms. Default: `30000`
 |`KAFKA_CLUSTERS_0_NAME` | Cluster name
 |`KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS` 	|Address where to connect 
-|`KAFKA_CLUSTERS_0_ZOOKEEPER` 	| Zookeeper service address 
```

Comment:
I would also add add deprecated flag to `ClusterStats.zooKeeperStatus` in kafka-ui-api.yaml